examples/greet.yaml

```diff
@@ -6,15 +6,15 @@ metadata:
     deislabs.io/wascc-action-key: MB4OLDIC3TCZ4Q4TGGOVAZC43VXFE2JQVRAXQMQFXUCREOOFEKOKZTY2
```

Comment:
this example appears to use the WasCC provider. should this be removed, or maybe a new example provided for the WASI provider instead?

Reply:
I would prefer a wasi and wascc example